### DISCUSSION TITLE
didn't understood ? 🥲

### DISCUSSION THREAD

**Discussion by parshvJS** [community/community#131623] [GitHub Education] [answered]
### Hi everyone,

Hey Everyone,
i have went to github student pack landing page to see what they offer but i didn't understood that much , what are free benefits of this pack.

**Comment by davevad93** [comment, marked as answer]
Hi @parshvJS ,

The Student Developer Pack offers a lot of features and services for free, first of all you get access to GitHub Pro for free which comes with a `Copilot` subscription, then you can access much more, I only list a few because the list is very long:

- A free JetBrains subscription for students, to be renewed annually.
- Free access to 25+ Microsoft Azure cloud services plus $100 in Azure credit. For students aged 18+.
- A Heroku credit of $13 USD per month for 24 months.
- Free GitKraken Pro license, enabling Pro access to GitLens and the entire GitKraken DevEx platform while you’re a student.
- Name.com free .LIVE domain name, free privacy protection, and a free SSL certificate.
- Digital Ocean $200 in platform credit for 1 year.
- Free 6-months access to all FrontEnd Masters courses and workshops.

As I said these are just a few of the features that the Student Developer Pack offers. 

Here are listed the requirements to get access to it:

- [Apply to GitHub Education as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student)

**Comment by parshvJS** [comment]
where are all this info didn't even knew one ,this is a lot,thanks for reply 😊✌️

**Reply by davevad93** [reply]
You're welcome @parshvJS , you can see the full list [here](https://education.github.com/pack/offers) after you've successfully applied.
